tinygradient.d.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * TinyGradient 0.4.0
+ * Copyright 2014-2018 Damien "Mistic" Sorel (http://www.strangeplanet.fr)
+ * Licensed under MIT (http://opensource.org/licenses/MIT)
+ */
+
+declare var tinygradient: tinygradient;
+
+interface TinyGradient {
```

Comment:
doesn't the interface needs to be exported (as well as the types) ?

Reply:
Yes, my product has belong code.

```
interface gradationSetting {
	high_filter: number,
	low_filter: number,
	high_color: string,
	low_color: string,
	grad?: TinyGradient;
};
const calcGradation = (p: number, setting: gradationSetting): string => {
	if (setting.grad === undefined) {
		setting.grad = tinygradient([setting.low_color, setting.high_color]);
	}
	...
}
```